core/node/rpc/service.go

```diff
@@ -56,8 +58,10 @@ type Service struct {
 	storage         storage.StreamStorage
 
 	// Streams
-	cache       *StreamCache
-	syncHandler river_sync.Handler
+	cache   *StreamCache
+	sync    riversync.Handler
+	syncv3  riversyncv3.Service
+	v3Syncs *xsync.Map[string, struct{}]
```

Comment:
Please use better naming than `sync`, `syncv3` and `v3Sync` for fields.

Reply:
Renamed. Not sure if new names are better tho :)